dalite_xblock/dalite_xblock.py

```diff
@@ -143,6 +153,30 @@ def is_lti_ready(self):
         """Check if this XBlock has all settings so it can connect to the LTI."""
         return all((self.launch_url, self.question_id, self.assignment_id))
 
+    def _render_student_view(self, context, in_studio):
+
+        if in_studio:
+            message = _("No question selected. Please click \"Edit\" and enter the assignment ID and question ID.")
+        else:
+            message = _("This task is not configured yet. You can't finish it right now.")
+
+        fragment = super(DaliteXBlock, self).student_view(context)
+        fragment.add_javascript(loader.load_unicode('public/js/dalite_xblock.js'))
+        fragment.initialize_js('DaliteXBlock')
+
+        if not self.is_lti_ready:
+            fragment.content = u''
+            context.update(self._get_context_for_template())
+            context.update({
+                "message": message
+            })
```

Comment:
@jbzdak Two suggestions:

1. "Invalid LTI Passport found." should instead be "No valid LTI passport found"
1. The passport format should say  "<name>:dalite-xblock:<long hash string>" and perhaps include an example like "mydalite:dalite-xblock:4o87qnyco78wyarcno8wycra8o7byvf87acn"